Modules/modExtendSaveAsText.ACM

```diff
@@ -27,7 +27,7 @@ Private Type AccObject
   blnAppended As Boolean 'Indicating whether or not ithe object has been appended to its parent yet
 End Type
 
-Private Type Ignore
+Private Type IGNORE

```

Comment:
Is this capilised for a reason? I this Reddick convention would be `Ignore`

Reply:
Changed